addons/website_sale/views/templates.xml

```diff
@@ -555,7 +555,7 @@
     </template>
 
     <template id="base_unit_price" name="Product Bas eunit price">
-        (<span class="o_base_unit_price" t-field="product.base_unit_price" t-options="{'display_currency': website.currency_id}"/>
+        (<span class="o_base_unit_price" t-esc="round(combination_info['base_unit_price'],2)" t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
```

Comment:
No need to round anymore, the widget will handle it for us.
There if a currency uses more than 2 digits we lose them on the widget.